src/EditorFeatures/TestUtilities2/Intellisense/ModernCompletionTestState.vb

```diff
@@ -18,6 +18,8 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
 
         Private Const timeoutMs = 10000
         Friend Const RoslynItem = "RoslynItem"
+        Private Shared ReadOnly ResponsiveCompletionThreshold As String = NameOf(ResponsiveCompletionThreshold)
+        Private Shared ReadOnly ResponsiveCompletionThresholdOption As EditorOptionKey(Of Integer) = New EditorOptionKey(Of Integer)(ResponsiveCompletionThreshold)
```

Comment:
Can't you just get this from the editor API itself? DefaultOptions.ResponsiveCompletionThresholdOptionId or something?